services/web/server/src/simcore_service_webserver/application_settings_utils.py

```diff
@@ -109,7 +109,9 @@ def convert_to_app_config(app_settings: ApplicationSettings) -> dict[str, Any]:
             else 0,
         },
         "smtp": {
-            "sender": getattr(app_settings.WEBSERVER_EMAIL, "SMTP_SENDER", None),
+            "sender": getattr(
+                app_settings.WEBSERVER_EMAIL, "SMTP_SENDER", None
+            ),  # SMTP_SENDER deprecated
```

Comment:
why was. this not removed from here?

Reply:
These utils are used to map the old configs with the env-vars. This was done to transition from the old (configfile-based) system to the new (env-vars/settings-based). Many tests depend on this so I did not want to modify too much at once.